### DISCUSSION TITLE
Is Renovate able to do individual package scheduling in the same group?

### DISCUSSION THREAD

**Discussion by lyonlai** [renovatebot/renovate#32208] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

renovate/renovate:38.133

### Please tell us more about your question or problem

From the doc [`schedule`](https://docs.renovatebot.com/configuration-options/#schedule) is for branch level creation. 

I thought Renovate will skip the not-scheduled package and create a branch for the others.

The situation I'm hitting is that I have a package `awscli` that updates very frequent that cause noises. But to reduce the number of branches & PRs it's grouped with other packages into common `major` and `non-major` group.  The group schedule is a 6 hour window everyday.  After I set the schedule for `awscli` the whole group is following the weekly schedule now.  

Besides putting `awscli` into its own group, is there any config option that let me do it but I missed?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
No, schedule determines branch creation so is a branch-level concept only. I'm open to future ideas of how this could work though. We'd have to weigh up if it would create more confusion or code complexity than it's worth.
